pages/common/pickle.md

```diff
@@ -0,0 +1,20 @@
+# pickle
+
+> A PHP extension installer based on Composer.
+> More information: <https://github.com/FriendsOfPHP/pickle>.
+
+- Install a specific PHP extension:
+
+`pickle install {{memcache}}`
```

Comment:
```suggestion
`pickle install {{extension_name}}`
```

Reply:
I put `memcache` as a definitive example. But it does look a bit odd. 🤔

In PHP, extension names can be referenced as:
- `memcache`
- `php-memcache` (won't work here, but used in package managers)
- `ext-memcache` (won't work here, but used in Composer)

So I thought having an example would be clearer.

Reply:
Maybe something like this? :thinking:

    - Install a specific PHP extension (don't include `php-` or `ext-` in the name):

    `pickle install {{extension_name}}`

Reply:
Using a concrete name definitely encouraged and in the spirit of tldr pages. I don't see any issues with it.